CHANGELOG.md

```diff
@@ -24,7 +24,13 @@ Adding a requirement of a major version of a dependency is breaking a contract.
 Dropping a requirement of a major version of a dependency is a new contract.
 
 ## [Unreleased]
-[Unreleased]: https://github.com/atlassian/jira-actions/compare/release-3.16.1...master
+[Unreleased]: https://github.com/atlassian/jira-actions/compare/release-3.17.0...master
+
+## [3.17.0] - 2022-04-13
+[3.17.0]: https://github.com/atlassian/jira-actions/compare/release-3.16.2...release-3.17.0
+### Added
+- Add new action View Focused Comment
+- Add new action View History Tab
```

Comment:
Do we want to create issues in JPERF project for that? I don't think it's necessary, but may explain to some readers why we are adding those.

Reply:
I'll stick to the previous convention.

Reply:
Because this is not necessary I'm closing this conversation.